Sources/update-strings/Commands/SortCommand.swift

```diff
@@ -27,15 +30,35 @@ struct SortCommand: ParsableCommand {
         
         print("  Found \(stringsFileURLs.count)")
         
+        var unsortedStringPaths: [String] = []
+        
         for stringsFileURL in stringsFileURLs {
             let relativePath = stringsFileURL.path
                 .replacingOccurrences(of: currentDirectoryURL.path, with: "")
                 .trimmingCharacters(in: CharacterSet(charactersIn: "/"))
             
-            print("Sorting \(relativePath)")
+            let currentStrings = try String(contentsOf: stringsFileURL)
             
             let stringsFile = try StringsFile(url: stringsFileURL)
-            try stringsFile.description.write(to: stringsFileURL, atomically: true, encoding: .utf8)
+            let newStrings = stringsFile.description
+            
+            guard currentStrings != newStrings else { continue }
+            
+            print("\(dryRun ? "Validating" : "Sorting") \(relativePath)")
```

Comment:
This doesn't really need different formatting, `--dry-run` commands generally look the same as running it normally.
```suggestion
            print("Sorting \(relativePath)")
```